projects/bbb_canbus_listener/main.cpp

```diff
@@ -203,9 +203,15 @@ int main(int argc, char **argv) {
 
                 std::map<std::string, NetworkTable::Value> values;
                 NetworkTable::Value gps_longitude;
-                double longitude = GET_GPS_LONG(frame.data);
+
+                // Perform a unit conversion from decimal degrees minutes to degrees.
+                double longitudeDDM = GET_GPS_LONG(frame.data);
+                int longitudeDD = longitudeDDM / 100;
+                double longitudeM = longitudeDDM - longitudeDD*100;
```

Comment:
Also should this be addition?

Reply:
Referenced from Tyler:
lat_degrees = int(lat/100)
lat_minutes = lat - lat_degrees*100
lat_double = lat_degrees + lat_minutes/60